python/paddle/fluid/tests/unittests/mkldnn/test_reshape_mkldnn_op.py

```diff
@@ -0,0 +1,136 @@
+#   Copyright (c) 2019 PaddlePaddle Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from __future__ import print_function
+
+import unittest
+import numpy as np
+import paddle.fluid.core as core
+
+from paddle.fluid.tests.unittests.op_test import OpTest
+from paddle.fluid.tests.unittests.test_reshape_op import TestReshapeOp, TestReshapeOpDimInfer1, TestReshapeOpDimInfer2, TestReshapeOpWithInputShape
+
+
+class TestMKLDNNReshape(TestReshapeOp):
+    def setUp(self):
+        self.init_data()
+        self.op_type = "reshape2"
+        self.use_mkldnn = True
+        self.initInput()
+        self.inputs = {"X": self.input_data}
+        self.attrs = {"shape": self.new_shape}
+        self.outputs = {
+            "Out": self.inputs["X"].reshape(self.infered_shape),
+            'XShape': np.random.random(self.ori_shape).astype("float32")
+        }
+
+    def test_check_grad(self):
+        pass
```

Comment:
why pass?

Reply:
We only need forward mkldnn reshape for INT8 inference, so only reshape fwd has been supported in this PR, we can consider to enable it if it's needed in the future.

Reply:
Hi. @xiaolil1  Do you plan to finish this PR and merge it? How is the status of this PR now?  Thank you~

Reply:
> Hi. @xiaolil1 Do you plan to finish this PR and merge it? How is the status of this PR now? Thank you~

Sure, this PR was blocked by #16462 and not the first priority of us before, new update will be submitted by this week, thanks.